DockerfileNEW

```diff
@@ -0,0 +1,80 @@
+# Stage: base image
+FROM node:18.12-bullseye-slim as base
```

Comment:
Currently, uses builder, will the switch to base impact? I guess not due to it being refactored to build from base?

`FROM node:18.12-bullseye-slim as builder`